subsys/net/l2/wifi/Kconfig

```diff
@@ -93,3 +93,31 @@ module-str = Log level for Wi-Fi Network manager module
 module-help = Enables using the Wi-Fi Network managers to manage the Wi-Fi network interfaces.
 source "subsys/net/Kconfig.template.log_config.net"
 endif # WIFI_NM
+
+config WIFI_MGMT_AP_STA_INACTIVITY_TIMEOUT
+	int "Station inactivity timeout in seconds"
+	default 300
+	help
+	  Station inactivity timeout is the period for which AP may keep a client
+	  in associated state while there is no traffic from that particular
+	  client. If a non-zero value is set, AP may choose to disassociate the
+	  client after the timeout.
+
+config WIFI_MGMT_AP_STA_SKIP_INACTIVITY_POLL
+	bool "Skip inactivity poll for stations after the inactivity timeout"
+	default n
+	help
+	  Skip inactivity poll for stations after the inactivity timeout. This
+	  will prevent the AP from sending null data frames to the stations after
+	  the inactivity timeout. This is useful to disconnect idle stations even
+	  if they are within the range of the AP.
+	  Note: This is only build time parameter, runtime configuration not supported.
+
+config WIFI_MGMT_AP_MAX_NUM_STA
+	int "Maximum number of stations allowed in station table"
+	range 1 4
```

Comment:
```suggestion
	range 1 2007
```

Reply:
I am fine with not having this change and limiting to 4, but it's easier to understand 2007 than 4.